test/utils/deploy-funded-account.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import { readFileSync } from "fs";
+import path from "path";
+import { StarknetPluginError } from "../../src/starknet-plugin-error";
+import { DEVNET_URL } from "../constants/constants";
+import { hardhatStarknetDeployAccount, hardhatStarknetNewAccount } from "./cli-functions";
+import { ensureEnvVar } from "./utils";
+
+export async function deployFundedAccount(url = DEVNET_URL) {
+    const network = ensureEnvVar("NETWORK");
+    const accountDir = ensureEnvVar("ACCOUNT_DIR");
+
+    if (network !== "devnet") {
```

Comment:
Could be an assertion

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/251/commits/f869821c5c5153f5e4b8878ca852a3fce03469ad